README.md

```diff
@@ -52,28 +50,38 @@ Tested with the following:
 
 * `letsencrypt_domain` - Domain the certificate is for.
 * `letsencrypt_email` - Your email as certificate owner.
+* `letsencrypt_tos_sha256` - Provide the SHA256 hash of the latest Terms-of-Service you agreed to
```

Comment:
I'd still like this default to be enabled in the defaults. The rationale is that the role should make things *as easy as possible to use*. Requiring the user to add extra variables to their role that could be provided as defaults IMHO is going backwards. The current role already has `--agree-terms` etc. The readme can clearly state which ToS the user will be accepting by using the role, and note that the user can change which ToS is being accepted via the variables.